data/lib/src/datasource_impl/received_share_datasource_imp.dart

```diff
@@ -60,4 +67,28 @@ class ReceivedShareDataSourceImp extends ReceivedShareDataSource {
       });
     });
   }
+
+  @override
+  Future<List<DownloadTaskId>> downloadReceivedShares(List<ShareId> shareIds, Token token, Uri baseUrl) async {
```

Comment:
I don't think we need baseUrl and token since we already logged in (cookie session) @manhduc-nguyen @hoangdat can you guys confirm?

Reply:
hmm, this PR focus to `sort`, what is the problem with `download` @dab246

Reply:
Ah it was my commits tho, but the question stills needs to be answered for later

Reply:
Probably because I have not synchronized the latest code from the master.